go.mod

```diff
@@ -7,23 +7,22 @@ require (
 	github.com/Microsoft/go-winio v0.4.15
 	github.com/aws/aws-sdk-go v1.35.24
 	github.com/blang/semver v3.5.1+incompatible
-	github.com/containerd/containerd v1.4.12
 	github.com/containerd/ttrpc v1.0.2
 	github.com/containerd/typeurl v1.0.2
-	github.com/docker/distribution v2.8.0+incompatible // indirect
-	github.com/docker/docker v20.10.12+incompatible
+	github.com/docker/distribution v2.8.1+incompatible // indirect
+	github.com/docker/docker v20.10.17+incompatible
```

Comment:
Do we have to change this? k/k is pinned on v20.10.12 - https://github.com/kubernetes/kubernetes/blob/6aa5efb/go.mod#L342

Reply:
yes, we need this, the older version of docker/distribution + docker/docker will drag containerd back in.